src/components/AccountDrawer/AnalyticsToggle.tsx

```diff
@@ -0,0 +1,20 @@
+import { t } from '@lingui/macro'
+import { analyticsOptOutToggle } from 'components/analytics/TraceAnalytics'
+import { useAtom } from 'jotai'
+
+import { SettingsToggle } from './SettingsToggle'
+
+export function AnalyticsToggle() {
+  const [allowAnalytics, updateAllowAnalytics] = useAtom(analyticsOptOutToggle)
+
+  return (
+    <SettingsToggle
+      title={t`Allow analytics`}
+      description={t`We use anonymized data to enhance your experience with Uniswap Labs products.`}
+      isActive={allowAnalytics}
+      toggle={() => {
+        updateAllowAnalytics(!allowAnalytics)
+      }}
```

Comment:
This may cause race conditions if you double-click it. Instead, negate the current value - as opposed to the value in closure:

```suggestion
      toggle={() => voidupdateAllowAnalytics((value) => !value)}
```